frontend/src/pages/Homepage/index.tsx

```diff
@@ -0,0 +1,312 @@
+import React, { FC, useState } from 'react';
+import { theme } from '../../common/theme';
+import StyleWrapper from '../../components/StyledWrapper';
+import { Wrap, Section, Img, Grid1, Grid2 } from './styled';
+import { Input } from '../../components/Input';
+import { H1, H3, LightText, LinkRegular } from '../../components/Typography';
+import s1Img from '../../../public/images/s1Img.jpg';
+import s2Img from '../../../public/images/s2Img.jpg';
+import s3Img from '../../../public/images/s3Img.jpg';
+import VylukaLogo from '../../../public/images/vyluka_logo.png';
+import MSMT from '../../../public/images/msmt1.png';
+import CeskoDigital from '../../../public/images/cd1.png';
+import CS from '../../../public/images/cs1.png';
+import Vodafone from '../../../public/images/vodafone1.png';
+
+const index = () => {
+  const clipPathImg = (
```

Comment:
tyhle definice svg mužeš mít nad `const Index = () => {`

Já tohle ve svým merge requestu předělával, aby nebylo potřeba svg takhle vkládat k tomu aby byly inline. Mužeš na to kouknout jak to mam použitý

Reply:
navíc to tu teď máš duplicitně, všechny tyhle svg máš jako soubory a pak ještě inline v kódu. Určitě se nepoužívají obě varianty